src/lib/components/FloatingArrow/FloatingArrow.test.svelte.ts

```diff
@@ -17,4 +18,58 @@ describe('FloatingArrow', () => {
 			expect(component).toBeInTheDocument();
 		}),
 	);
+
+	it(
+		'renders position based on context placement',
+		withEffect(() => {
+			const arrowRef = document.createElement('div');
+			const floating = useFloating({ placement: 'left' });
+			const { getByTestId } = render(FloatingArrow, {
+				props: { ref: arrowRef, context: floating.context, width: 20, height: 20 },
+			});
+			const component = getByTestId('floating-arrow');
+			expect(component.style.left).toBe('calc(100% - 0px)');
```

Comment:
Be sure to call `cleanup` imported from `@testing-library/svelte` at the end of every test.
https://testing-library.com/docs/svelte-testing-library/api#cleanup
I just read that if you import something in the setup file this is done automatically.
Let's make sure cleanup is done properly for this PR and then automate this process in another?

Reply:
Thanks for the reminder, this has been done.